examples/example-gcp-observability/README.md

```diff
@@ -0,0 +1,39 @@
+gRPC GCP Observability Example
+================
+
+The GCP Observability example consists of a Hello World client and a Hello World server instrumented for logs, metrics and tracing. 
+
+__Please refer to GCP observability user guide for setting up authorization to acess Google Cloud Platform with Google user credentials.__
+
+### Build the example
+
+Build the Observability hello-world example client & server. From the `grpc-java/examples/examples-gcp-observability`
+directory:
+```
+$ ../gradlew installDist
+```
+
+This creates the scripts `build/install/example-gcp-observability/bin/observability-hello-world-client` and
+`build/install/example-gcp-observability/bin/observability-hello-world-server`.
+
+### Run the example with configuration
+
+To use Observability, you should first setup and configure authorization and credentials to use gCloud CLI. 
```

Comment:
Do you need to "setup and configure authorization and credentials to use gCloud CLI" or "setup and configure authorization and credentials to **_using_** gCloud CLI" ? Just trying to understand what is being conveyed here

Reply:
Updated, PTAL